samples/interactive-tutorials/setup/delete-events-bigquery-table.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main() {
```

Comment:
Is this a sample? We don't have region tags, or a lot of the normal strucutre.

Reply:
it is not a sample, this is the supporting code for the uploading the catalog data